### ISSUE TITLE
Arithmetic operation not working correctly

### ISSUE THREAD

**Issue by pranay001** [navinsubramani/LabVIEW-Script-Language#25]
variable = 1 - 2 - 6
print(variable)

for the above function we are getting the result as +5, but it has to be -5.

**Comment by navinsubramani** [comment]
Yes, this a bug and we will not handle now. Let this be in the backlog. We can use the brackets to make the function work properly. The reason for the issue is the arithmetic operation happens from the right to left so (2-6) is done first and then 1-(-4) is done next. We can avoid this by using (1-2)-6
